### PR TITLE
chore: Review updates

### PR DESCRIPTION
**Pull request by alanbsmith**
## Summary

Fixes: #1234

This PR:
- Updates styles to use our new style utils
- Updates docs to use our doc utils
- Updates tests to (you guessed it) use our test utils

## Release Category

Components

---

## Checklist

- [ ] MDX documentation adheres to Canvas Kit's [Documentation Guidelines](https://workday.github.io/canvas-kit/?path=/docs/guides-documentation-guidelines--page)
- [ ] Label `ready for review` has been added to PR

## For the Reviewer

- [ ] PR title is short and descriptive
- [ ] PR summary describes the change (Fixes/Resolves linked correctly)
- [ ] PR Release Notes describes additional information useful to call out in a release message or removed if not applicable
- [ ] Breaking Changes provides useful information to upgrade to this code or removed if not applicable

## Areas for Feedback? (optional)

- [ ] Code
- [ ] Documentation
- [ ] Testing
- [ ] Codemods

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -55,7 +55,6 @@ module.exports = {
     '@typescript-eslint/member-delimiter-style': 'off',
     '@typescript-eslint/type-annotation-spacing': 'off',
     '@emotion/jsx-import': 'error',
-    '@emotion/no-vanilla': 'error',
```
Comment (alanbsmith) [inline .eslintrc.js:58]: This is a change that will be reflected in the latest version of canvas-kit, but it's not in this branch yet (and I needed it).

File: modules/preview-react/loading-sparkles/lib/LoadingSparkles.tsx [hunk 2]
```diff
@@ -41,56 +43,56 @@ const LOADING_ANIMATION = keyframes({
...
+};
 
-/**
- * A simple container for the loading sparkles.
- */
-const Container = styled('div')({
+const loadingSparklesStyles = createStyles({
   display: 'inline-flex',
-  gap: '1px',
+  gap: px2rem(1),
```
Comment (jimcasey) [inline modules/preview-react/loading-sparkles/lib/LoadingSparkles.tsx:81]: 💯

File: modules/preview-react/loading-sparkles/lib/LoadingSparkles.tsx [hunk 3]
```diff
@@ -41,56 +43,56 @@ const LOADING_ANIMATION = keyframes({
...
-);
+  '&:nth-child(1)': {
+    animationDelay: '0ms',
+  },
+  '&:nth-child(2)': {
+    animationDelay: `calc(${ANIMATION_DURATION_MS}ms * (1/3))`,
+  },
+  '&:nth-child(3)': {
+    animationDelay: `calc(${ANIMATION_DURATION_MS}ms * (2/3))`,
+  },
```
Comment (jimcasey) [inline modules/preview-react/loading-sparkles/lib/LoadingSparkles.tsx:63]: 👍 Clever!

**Review by jimcasey — APPROVED** [review]
🐑

**Pull request merged**